src/components/navigation/nav.module.scss

```diff
@@ -108,10 +113,10 @@ a {
 .nav {
   z-index: 10;
   position: fixed;
-  top: 80px;
+  top: 45px;
   overflow-y: auto;
   width: 248px;
-  height: calc(100% - 80px);
+  height: 100vh;
```

Comment:
by this ways: make sure the side navbar is scrollable other wise, we can't see bottom messages on mobile screens
![image](https://github.com/atlp-rwanda/Team-Sostene-E-commerce-fe/assets/91186046/1193b97e-0dc1-4d9e-ad32-8532d46be37c)
![image](https://github.com/atlp-rwanda/Team-Sostene-E-commerce-fe/assets/91186046/31c5f3da-e043-4aae-8dc6-12aab569ea18)
![image](https://github.com/atlp-rwanda/Team-Sostene-E-commerce-fe/assets/91186046/9d4c1165-fe92-4f7f-89f8-63e9a2a6005a)
You can even scroll inside sideNav